src/components/custom/dashboard/client/pages/diet-tracker/MealDetailModal.jsx

```
@@ -0,0 +1,346 @@
+            </div>
+
+            {/* Macro Distribution Chart */}
+            <div className="space-y-4">
+              <h4 className="text-zinc-300 font-medium">Distribution</h4>
+
+              <div className="relative w-32 h-32 mx-auto">
+                <svg
+                  className="w-32 h-32 transform -rotate-90"
+                  viewBox="0 0 36 36"
+                >
+                  <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)